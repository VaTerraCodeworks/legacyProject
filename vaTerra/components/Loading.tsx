import { View, ActivityIndicator, StyleSheet } from 'react-native';
import React from 'react';

export default function Loading() {
  return (
    <View style={styles.container}>
      <ActivityIndicator size="small" color="#009c97" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
});
