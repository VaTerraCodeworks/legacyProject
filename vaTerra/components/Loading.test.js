import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './Loading';

test('The Loading screen has not changed', () => {
  const loadingScreen = renderer.create(<Loading />).toJSON();
  expect(loadingScreen).toMatchSnapshot();
  expect(loadingScreen.props).toEqual({ style: { flex: 1 } });
});
