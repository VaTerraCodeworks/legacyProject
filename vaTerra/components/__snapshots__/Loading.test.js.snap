// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loading screen has not changed 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ActivityIndicator
    color="#009c97"
    size="small"
  />
</View>
`;
